_dashboards/index.md

```diff
@@ -11,15 +11,23 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# What is OpenSearch Dashboards?
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is an open-source analytics and search application that gives you a view of information and data all in one view. OpenSearch Dashboards allows you to monitor events, make decisions, inform others, and see trends and displays multiple visualizations that work together on a single screen. These visuals give you a comprehensive view of your data and provide key insights for at-a-glance decision-making.
 
+![Visualization example image](../images/what-is-dashboards.jpg)
 
-## Get started with OpenSearch Dashboards
+OpenSearch Dashboards allows you to create dashboards for many reasons, and many types of dashboards. OpenSearch Dashboards lets you to do the following:  
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+* See-all in one view-the data you need to make informed decisions.
```

Comment:
```suggestion
* See an all-in-one view of the data you need to make informed decisions.
```

Reply:
just a note, pls make commit separately from Github (don't click "Commit suggestion" button), it will create a non-valid DCO signature.